azuread/README.md

```diff
@@ -0,0 +1,41 @@
+# Azure AD / Entra ID App Function HTTP Trigger Example
+This C# script example demonstrates how to call the SGNL Access Service from an Azure App Function HTTP Trigger. This app function and trigger is then configured as part of an custom authentication extension.
+
+
+# Prerequisites
+
+1. You have a SGNL environment up and running.
+ 
+2. You have configured connectors to ingest data and have defined the appropriate relationship mappings.
+ 
+3. You have created policy snippets and policy for evaluation.
+ 
+4. You have created an protected application to test. You can test with this [example](https://jwt.ms/) provided by Microsoft.
+ 
+5. You will need Git to clone the example repository. You can follow the steps to install Git [here](https://github.com/git-guides/install-git).
+
+
+See our [Help Guides](https://support.sgnl.ai) for steps on configuring data sources and policies.
+
+
+## Steps For Configuring the custom authentication extension.
```

Comment:
nitpick: your capitalization on your headings is inconsistent - I'm not partial to which way you go, but could you update them all to either sentence or title case?

Reply:
Got it! I will update to title case.